pyiron_atomistics/atomistics/job/atomistic.py

```diff
@@ -394,7 +394,10 @@ def db_entry(self):
                 parent_structure = self.structure.get_parent_basis()
             else:
                 parent_structure = self.structure.copy()
-            db_dict["ChemicalFormula"] = parent_structure.get_chemical_formula()
+            chem_formula = parent_structure.get_chemical_formula()
+            if len(chem_formula) > 30:
+                chem_formula = chem_formula[:29]+"+"
```

Comment:
My concern with leaving any of the real formula intact is that then certain parsing operations will silently return the wrong results. e.g. imagine that our limits were much shorter and we would up with something like "Fe302Mn1000" -> "Fe302Mn10" (wrong count), or "O1234P56He" -> "O1234P56H" (wrong species).

I would suggest instead a value like "OVERFLOW_ERROR", and let the user exploit pyiron tables to extract the chemistry directly from the stored structures.